go.mod

```diff
@@ -1,11 +1,67 @@
 module github.com/privacybydesign/uniqueid-issuer
 
-go 1.16
+go 1.18
 
 require (
 	github.com/go-chi/chi v4.1.2+incompatible
 	github.com/go-chi/cors v1.0.0
-	github.com/go-errors/errors v1.4.1 // indirect
 	github.com/privacybydesign/irmago v0.9.1-0.20220201194040-b00bf11ce3b8
```

Comment:
Maybe it's nice to update irmago to the latest version then as soon as we release it.